tests/compiler/std/symbol.optimized.wat

```diff
@@ -638,29 +640,34 @@
     br $while-continue|0
    end
   end
-  local.get $4
-  local.tee $2
+  local.get $5
+  local.tee $3
   local.get $0
+  local.tee $2
   i32.load
   i32.ne
   drop
-  local.get $0
   local.get $2
+  local.get $3
   i32.store
-  local.get $0
+  local.get $2
   local.get $1
   i32.store offset=4
-  local.get $0
+  local.get $4
+  local.tee $1
+  local.get $2
+  local.tee $3
   i32.load offset=8
+  i32.ne
```

Comment:
While not adding much, dropping an `i32.ne` seems trivial, and I'd expect this to be dropped in any case.